### PR TITLE
Classic sql injection

### LINKED ISSUE #54: Add classic (feedback-based) SQL injection
### Acceptance Criteria

1. User must be able to perform SQL injection and receive output that either contains the query being executed or shows results from the database.
1. Test must be created to demonstrate this vulnerability automatically.
1. Video must be created to demonstrate this vulnerability manually.
1. Video must be stored in `/docs/vulnerabilities/sql-injection/`.

### PR DESCRIPTION
**Pull request by tanner-g**
Closes #54

### CODE REVIEW

**Comment by tanner-g** [comment]
Saving progress. Not ready yet.

**Comment by tanner-g** [comment]
It looks like it should be vulnerable to SQL injection... but it doesn't quite work yet.

**Comment by JasonPuglisi** [comment]
Converted video to MP4. Amended commit and force pushed to preserve ownership.

**Comment by tanner-g** [comment]
* pulled in changes
* added in sql injection test missing from test_db
* working on wiki edits needed

File: tests/base/test_db.py [hunk 1]
```diff
@@ -108,3 +108,13 @@ def test_save_video_db(video_id, user_id, extension, success):
     """ Tests save_video_db() functionality """
     assert save_video_db(video_id, extension, user_id) == success
     assert delete_video_db(video_id) == success
+
+@pytest.mark.parametrize('username, response', [
+    (TEST_USERS[0][0], [TEST_USERS[0][0]]),
+    ('FakeUser', ['[X] User Not Found']),
+    ('', ['[X] User Not Found']),
+    ])
```
[the remark this answers is not in the document]
Comment (tanner-g) [reply tests/base/test_db.py]: Added!

**Pull request merged**